eng/Versions.props

```diff
@@ -252,7 +252,7 @@
   <!-- Workload manifest package versions -->
   <PropertyGroup>
     <AspireFeatureBand>8.0.100</AspireFeatureBand>
-    <AspireWorkloadManifestVersion>8.0.0-preview.1.23557.2</AspireWorkloadManifestVersion>
+    <AspireWorkloadManifestVersion>8.0.0-preview.3.24060.4</AspireWorkloadManifestVersion>
```

Comment:
This property doesn't get auto-updated as it doesn't follow the naming conventions. It should be the same as the value in Version.Details.xml (without the `.` and `-` (I think).

Reply:
This will cause frustration in future dependency flow updates so we should probably fix this asap, probably with the next dependency flow.